demo/code-demo/step_authorize.html

```diff
@@ -1,43 +1,44 @@
 <!DOCTYPE html>
 <div class="line">
-    <h4>Sample Report</h4>
-    To Embed using a sample report, you can just move to next step.<br/>
-    <button id="nextStepWithSample" class="btn btn-primary" onclick="OpenEmbedStepWithSample()">Embed Sample Report</button><br/><br/><br/>
-    <a href="#" onclick="ToggleEmbedWithSpecificReportDiv()">Embed Your Own Report</a><br/>
+    <h4>Sample report</h4>
+    To embed using a report supplied with the sample, select <b>Embed sample report</b>.<br/>
+    <button id="nextStepWithSample" class="btn btn-primary" onclick="OpenEmbedStepWithSample()">Embed sample report</button><br/><br/><br/>
+    <a href="#" onclick="ToggleEmbedWithSpecificReportDiv()">Embed your own report</a><br/>
 </div>
 <div id="EmbedWithSpecificReportDiv" style="display: none;">
     <div class="line">
         <h4>Prerequisites</h4>
         <ul>
-            <li>A workspace in <a href="https://azure.microsoft.com/en-us/documentation/articles/power-bi-embedded-get-started/">Azure PowerBI Embedded Service.</a></li>
-            <li>PowerBI Report imported to your workspace.</li>
+            <li>A workspace in the <a href="https://azure.microsoft.com/en-us/documentation/articles/power-bi-embedded-get-started/">Power BI embedded service.</a></li>
```

Comment:
you write Power BI (as separated words) above, and PowerBI as one word below.... Please make sure it is consistent.